recipes/clustalo/build.sh

```diff
@@ -3,7 +3,12 @@
 export CPPFLAGS="-I$PREFIX/include"
 export LDFLAGS="-L$PREFIX/lib"
 
-./configure --prefix=$SRC_DIR OPENMP_CFLAGS='-fopenmp' CFLAGS='-DHAVE_OPENMP'
+if [ "$(uname)" == "Darwin" ]; then
+    # clang doesn't accept -fopenmp and there's no clear way around that
+    ./configure --prefix=$SRC_DIR
```

Comment:
Shouldn't prefix be the install path?
```bash
./configure --prefix=$PREFIX
```

Reply:
One would presume, but there's no `make install` so it doesn't really matter what you put there. Truth be told I was just lazy and never changed that from the original recipe, but you're correct that that should either be removed or changed to `$PREFIX` for the sake of clarity/consistency :)